### ISSUE TITLE
[android] Show the number of used / total  GPS satellites on the "my position" card

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#6484]
https://github.com/organicmaps/organicmaps/pull/6480#pullrequestreview-1713986100
Show the number of used / total GPS satellites and how old is the location after selecting the "my position" arrow

**Comment by pastk** [comment]
Its a good feature for more "advanced" users. And probably this info should be visible when a PP is opened only.
Also this info should be updated in real-time (I don't think we do update anything in real-time in PP atm).

**Comment by biodranik** [comment]
> I don't think we do update anything in real-time in PP atm

We do.

**Comment by biodranik** [comment]
Good point about the arrow. Let's make it simple: no additional UI panels/components. The location button can change to a satellite icon and display numbers until the app gets a GPS fix. Then the number of satellites is not important anymore.
